claasp/ciphers/block_ciphers/speck_block_cipher.py

```diff
@@ -126,25 +129,33 @@ def key_initialization(self, key_bit_size):
 
         return key_schedule, l_schedule
 
-    def round_function(self, p1, p2, key):
+    def round_function(self, p1, p2, key, tag=None):
```

Comment:
Not sure the tag should be defined at a round level. It should be defined at a component level.